### PR TITLE
Added the insert button in load image step

### LINKED ISSUE #644: Load Image step has no insert step button
### Please describe the problem (or idea)

> What happened just before the problem occurred? 

All the steps have an `+add` button which lets the user insert a step in betwee ln but the initial `load image` step does not.
There should be a button in that step too which lets the user add a step to the start of the sequence.

screenshot_20190110-181650

screenshot_20190110-181655

Please note: This is a screenshot from the `gh-pages` branch on sequencer.publiclab.org

### PR DESCRIPTION
**Pull request by Harshithpabbati**
Fixes #644 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` and `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Harshithpabbati** [comment]
@publiclab/is-reviewers

File: examples/lib/defaultHtmlStepUi.js [hunk 1]
```diff
@@ -128,8 +128,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
 
       
     }
-
-    if (step.name != "load-image") {
```
Comment (Divy123) [inline examples/lib/defaultHtmlStepUi.js]: Deleting the if condition here would result in delete option also available to load-image step which should not be the case.
Instead what you can do is simply pull the insert step button functionality out of that if condition and check if the number of steps involved is 1 then you can disable the button on the current step.
Does that make sense?

Comment (jywarren) [reply examples/lib/defaultHtmlStepUi.js]: Shall we take the next step here? Thanks so much!

**Comment by HarshKhandeparkar** [comment]
Branch deleted (i think)

**Comment by Harshithpabbati** [comment]
Yup !! I need to make many changes even.

**Pull request closed without merging**